Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 2
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 3
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 9
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 17
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 24
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 31
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 1
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 4
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 5
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 7
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 10
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 11
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 13
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 18
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 23
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 30
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 35
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 36
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 37
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 6
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 8
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 12
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 14
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 15
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 16
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 19
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 20
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 21
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 22
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 25
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 26
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 27
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 28
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 29
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 32
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 33
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 34
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 38
Traceback (most recent call last):
  File "forward_3d.py", line 15, in <module>
    sources = spyro.insert_fixed_value(lines, -0.10, 0)
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in insert_fixed_value
    tmp = [list(point) for point in points]
  File "/home/olender/cse23/spyro/spyro/utils/geometry_creation.py", line 95, in <listcomp>
    tmp = [list(point) for point in points]
TypeError: 'numpy.float64' object is not iterable
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 39
